src/library/scanner/libraryscanner.cpp

```diff
@@ -99,11 +99,11 @@ LibraryScanner::LibraryScanner(QWidget* pParentWidget, TrackCollection* collecti
 }
 
 LibraryScanner::~LibraryScanner() {
-    // A scan is running.
-    if (m_scannerGlobal) {
-        // Cancel any running library scan.
-        cancel();
-    }
+    // Wait until there is no pending scan request
```

Comment:
With this logic, don't we delay Mixxx closing too much ? What is the typical mutex lock time ?